namui/namui/src/namui/system/media/core/media_source.rs

```diff
@@ -0,0 +1,71 @@
+use std::{
+    path::{Path, PathBuf},
+    sync::Arc,
+};
+
+#[derive(Debug, Clone)]
+pub enum MediaSource {
+    Path(std::path::PathBuf),
```

Comment:
This path is for bundle or system drive path?

Reply:
It's system drive